audit_exceptions/universal_binary_allowlist.json

```diff
@@ -19,6 +19,7 @@
   "node-sass",
   "openjdk",
   "openjdk@11",
+  "passenger",
```

Comment:
Can we use
```ruby
deuniversalize_machos
```
to thin out the universal binaries instead?

Reply:
It seems it works, thanks!